pkg/api/output.go

```diff
@@ -143,13 +143,13 @@ func (instance *Instance) DisplayOutput() error {
 	return nil
 }
 
-// filterOutput filters the outputs that get printed
+// FilterOutput filters the outputs that get printed
 // first it fills out the Deprecated and Removed booleans
 // then it returns the outputs that are either deprecated or removed
 // and in the component list
 // additionally, if instance.OnlyShowDeprecated is true, it will remove the
 // apiVersions that are deprecated but not removed
-func (instance *Instance) filterOutput() {
+func (instance *Instance) FilterOutput() {
```

Comment:
I believe the original reasoning behind doing it this way has to do with the fact that the various output flags can change the behavior of the booleans. When we get the info to populate output, we don't want to have to take those into account.

The collection of information should be agnostic of what the user wants to see at the end, and then we filter it to the user's desired behavior.

Reply:
My attempt to rename `IsVersioned` to `PopulateOutputs` and move  the call to ``instance.FilterOutput` inside of `PopulateOutputs` almost worked, but it would break the Helm code which iterates Helm releases and adds to the outputs.

I think the best action is some wider assessment / refactoring to improve Pluto's usability as packages, but that probably shouldn't happen as part of this work (facilitating admission-controller using Pluto).

Reply:
This change alone is simple enough, and very safe, so I'm good with just merging this and doing any necessary refactoring in a later PR.